packages/react-sdk/src/tree/webstudio-component.tsx

```diff
@@ -57,13 +55,42 @@ export const WebstudioComponent = ({
     return <></>;
   }
   return (
-    <Component {...props}>
+    <Component {...props} ref={ref}>
       {renderWebstudioComponentChildren(children)}
     </Component>
   );
+});
+
+export const idAttribute = "data-ws-id" as const;
+export const selectorIdAttribute = "data-ws-parent-id" as const;
```

Comment:
its not just parent id, is it? I think its the path up to the root, so it would be data-ws-path and pathAttribute if I am correct

Reply:
or data-ws-selector